### DISCUSSION TITLE
How to build and compile ACE+TAO-7_1_2 with SSL support on Windows (VS2017 x86_32)?

### DISCUSSION THREAD

**Discussion by xpston008** [DOCGroup/ACE_TAO#2189] [Q&A]
How to build and compile ACE+TAO-7_1_2 with SSL support on Windows (VS2017 x86_32)? 
I haven't been able to find any relevant manuals or guides. Could you provide some assistance, please? Thank you.

`openssl-1.1.1q`

**Comment by mitza-oci** [comment]
Set the environment `SSL_ROOT` to the OpenSSL installation location.
Add `openssl11=1` and `ssl=1` to the MPC features (in one of its configuration files or using `-features` on the command line) and generate the solution with mwc.pl

**Comment by gnsh0802** [comment]
> How to build and compile ACE+TAO-7_1_2 with SSL support on Windows (VS2017 x86_32)? I haven't been able to find any relevant manuals or guides. Could you provide some assistance, please? Thank you.
> 
> `openssl-1.1.1q`

Any luck in resolving the issue ?

An example of building ACE+TAO with OpenSSL using vs2019 is here https://github.com/DOCGroup/ACE_TAO/actions/runs/7770892998/job/21191585321

This could help to solve your issue.
